src/test/kotlin/MainKtTest.kt

```diff
@@ -1,10 +1,87 @@
 import org.junit.Test
+import reposanalyzer.config.Language
+import reposanalyzer.methods.MethodSummary
+import reposanalyzer.methods.MethodSummaryStorage
+import java.io.File
 import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
 
 internal class MainKtTest {
 
+    private fun createMethodSummary(name: String, fullName: String, filePath: String) =
+        MethodSummary(name = name, fullName = fullName, filePath = filePath, language = Language.JAVA)
+
     @Test
-    fun testGetSum() {
-        assertEquals(4, getSum(2, 2))
+    fun methodStorageTest() {
```

Comment:
It's interesting to test all things in one function :)

Reply:
fixed